README.md

```diff
@@ -11,7 +11,7 @@ Azure Container Apps is built on the foundation of powerful open-source technolo
 
 Please visit the [FAQs wiki page](https://github.com/microsoft/azure-container-apps/wiki/FAQs).
 
-## Known Issues for Public Preview
+## Known Issues
 
 Please review the [known issues wiki page](https://github.com/microsoft/azure-container-apps/wiki/Known-Issues-for-public-preview) before reporting new issues.
```

Comment:
Please add new wiki page and replace the link "https://github.com/microsoft/azure-container-apps/wiki/Known-Issues-for-public-preview" with the new one

Reply:
I don't have the right to create new wiki page on this repository.
<img width="683" alt="image" src="https://user-images.githubusercontent.com/4566555/176355634-43b17b0c-8074-4034-a087-b21280618b86.png">

Who can create the page and manages known issues for GA?